efi/lds/elf_loongarch64_efi.lds

```diff
@@ -0,0 +1,166 @@
+OUTPUT_FORMAT("elf64-loongarch", "elf64-loongarch", "elf64-loongarch")
```

Comment:
is this already upstream in gnu-efi?

Reply:
Yes, LoongArch is already supported upstream of gnu-efi.
 https://sourceforge.net/p/gnu-efi/code/ci/master/tree/gnuefi/elf_loongarch64_efi.lds